src/components/Presentation/Dropzone.vue

```diff
@@ -35,7 +35,7 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-$stripeColor: #f0f0f3;
+$stripeColor: var(--v-dropzone-darken1);
```

Comment:
How does this work?

None of the modified dropzone colors are any of the colors listed at https://vuetifyjs.com/en/styles/colors so some mechanism is making up a whole new color palette (and not doing a good job, since lighten1 - lighten5 are all white).

I also think the new hover contrast is too high and doesn't look quite right, but I'm not sure how to deal with that other than adding another vuetify variable for "dropzoneHover".  I wonder if that may be the right thing to do, since whatever generated this palette produces step-sizes that are just too large to be useful to us. 

![Screenshot from 2019-11-01 09-31-03](https://user-images.githubusercontent.com/4214172/68028106-5ef03e00-fc8a-11e9-9a4f-cd04e6a93ea8.png)


![Screenshot from 2019-11-01 09-28-42](https://user-images.githubusercontent.com/4214172/68027984-26e8fb00-fc8a-11e9-9726-982f34e7b923.png)

Reply:
> How does this work? None of the modified dropzone colors are any of the colors listed

See my changes in `vuetifyConfig.js`. The colors I set as the `dropzone` value in both themes are already computationally modified from the color set.

> I also think the new hover contrast is too high and doesn't look quite right

I kind of felt the same way, but was trying to avoid having two different custom keys just for this one component. Maybe rather than using Vuetify's lighten and darken for the stripe color, we can do a SASS `darken` function to do one that is less of a dramatic step.

Reply:
It also occurs to me that we are firmly in the realm of design here now that we're talking about theming and how many and what things should be expressed in a theme. I'd like to hear from @jtomeck on that specific issue.

Reply:
> See my changes in vuetifyConfig.js. The colors I set as the dropzone value in both themes are already computationally modified from the color set.

 I don't think I understand. If `$vuetify.colors.dropzone` is ` colors.grey.lighten3` AKA `#EEEEEE` AKA `--v-dropzone-base`, what is `--v-dropzone-darken1`?

It would intuitively be the same as `colors.grey.lighten2`, but it isn't.  It's actually just `hex(colors.grey.lighten3) - 0x282828` which isn't anything.

> we can do a SASS darken function to do one that is less of a dramatic step.

How?  `--v-dropzone-darken1` is defined at runtime, and the app theme is defined at runtime.   Even if you could import the light and dark variants at build-time, I'm not sure how you'd swap between with a css-only approach.

Reply:
> It would intuitively be the same as colors.grey.lighten2, but it isn't.

I think the color transformations aren't perfectly uniform additive jumps in RGB space, they may occur in a different color space and change depending on a variety of parameters, with the intent of perceptual linearity.

> How? --v-dropzone-darken1 is defined at runtime, and the app theme is defined at runtime. Even if you could import the light and dark variants at build-time, I'm not sure how you'd swap between them with a css-only approach.

That's true, I forgot that this was using runtime CSS variables. It would need to be a CSS transform rather than a SASS transform, [but that does exist](https://topaxi.codes/modifying-css-colors-with-the-color-function/).